app/experimenter/experiments/tests/test_models.py

```diff
@@ -1352,10 +1352,12 @@ def test_allocate_buckets_generates_bucket_range(self):
         feature = NimbusFeatureConfigFactory(slug="feature")
         experiment = NimbusExperimentFactory.create_with_lifecycle(
             NimbusExperimentFactory.Lifecycles.CREATED,
+            is_rollout=False,
             application=NimbusExperiment.Application.DESKTOP,
             channel=NimbusExperiment.Channel.RELEASE,
             feature_configs=[feature],
             population_percent=Decimal("50.0"),
+            targeting_config_slug=NimbusExperiment.TargetingConfig.MAC_ONLY,
```

Comment:
If `is_rollout` is `False` then why do we need to add a targeting config slug here?

Reply:
Not in this one, since it's not a rollout

Reply:
I don't understand what you mean?

Reply:
we don't need to add targeting_config_slug here because it's not a rollout